src/IncrementalChecker.ts

```diff
@@ -128,6 +130,16 @@ class IncrementalChecker {
     return endsWith(filePath, '.d.ts') || linterExclusions.some(matcher => matcher.match(filePath));
   }
 
+  /**
+   * Check for vue-loader in user's project node_modules.
+   */
+  hasVueLoader(): boolean {
+    const projectPath = path.dirname(this.programConfigFile);
+    const contextResolution = Resolve.sync('vue-loader', { basedir: projectPath });
+    
+    return require(contextResolution) ? true : false;
```

Comment:
It seems you still have to try/catch this line and return false on error ...

Reply:
Oh right! Brain freeze! Of course that's the problem. LOL. Will fix...